src/components/Execution/Webhook.jsx

```diff
@@ -27,23 +32,23 @@ export const Webhook = ({ hook }) => {
         }
       }
 
-      await client.stackClient.fetchJSON(
-        'POST',
-        `/jobs/webhooks/${hook.id}`,
-        body
-      )
-      setIsWorking(false)
+      try {
+        await client.stackClient.fetchJSON('POST', hook.links.webhook, body)
```

Comment:
It seems to me we aggred about using `/jobs/webhooks/${hook.id}` instead of the `links` with the complete URL?

Reply:
Isn't creating the url by hand less resistant to possible future changes, rather than using what the cozy-stack gives us?

Reply:
Yes, although the probability of changing this API seems quite low. But I though you had issue with this method?

Reply:
The issue was with `hook.links.self`, which is not the webhook's URL but a JSON API information irrelevant to calling the webhook. `hook.links.webhook` works fine.

Reply:
All right then :ok_hand: